docs/content/quickstart.md

```diff
@@ -18,31 +18,27 @@ For Spin, follow [the Spin quickstart guide](https://spin.fermyon.dev/quickstart
 - clone and install Spin using cargo, or
 - clone and build Spin from source.
 
-### Bartholomew CLI
-
-For the `bart` CLI, there are two options:
-- download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/), or
-- clone and install `bart` from source, using the following commands:
-
-```bash
-$ git clone https://github.com/fermyon/bartholomew.git
-$ cd bartholomew
-$ make bartholomew
-```
-
 ## Templates
 
-This quickstart method uses a Bartholomew website template. So while we do require `spin` and `bart` CLI (as per the details above) everything else we need (to launch our Bartholomew CMS website) is packaged up in the Bartholomew website template. Including the `bartholomew.wasm` and the `spin_static_fs.wasm` files which take care of Bartholomew's business logic and [Spin's file server](https://github.com/fermyon/spin-fileserver) needs, respectively. We will start working with the template in the next section.
+This quickstart method uses a Bartholomew website template. So while we do require `spin` (as per the details above) everything else we need (to launch our Bartholomew CMS website) is packaged up in the Bartholomew website template. Including the `bartholomew.wasm` and the `spin_static_fs.wasm` files which take care of Bartholomew's business logic and [Spin's file server](https://github.com/fermyon/spin-fileserver) needs, respectively. We will start working with the template in the next section.
 
 ## Create Your First Website
 
 We can now generate a new repository with the same directory and file structure as the template. Simply visit [the template's location](https://github.com/fermyon/bartholomew-site-template) on [GitHub](https://github.com/fermyon/bartholomew-site-template). Then click on the green `Use this template` button and follow the prompts. Here are some additional details about [creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template) if you are interested.
 
 ![Use template](../static/image/docs/use-template.png)
 
-## Customize Your Website
+## Fetch Your Website
 
-You can clone and customize your own new codebase and then run the website locally, to suit your needs:
+The previous step creates a new repository in **your** GitHub account. This step fetches that new repository using the `git clone` command:
+
+```bash
+git clone git@github.com:your-username/your-repo-name.git
```

Comment:
Dunno which is preferred but you use the `https://github.com/.../...` elsewhere (I think the Spin docs use that too).  Unless there's a reason to use different formats?